src/lotus_json/actor_states/methods/evm_actor_params.rs

```diff
@@ -0,0 +1,158 @@
+                        }
+                    }
+                }
+            }
+        )+
+    };
+}
+
+impl_evm_constructor_params!(10, 11, 12, 13, 14, 15, 16);
+
+#[derive(Debug, Serialize, Deserialize, Clone, PartialEq, Eq, JsonSchema)]
+#[serde(rename_all = "PascalCase")]
+pub struct EVMDelegateCallParamsLotusJson {
+    #[schemars(with = "LotusJson<Cid>")]
+    #[serde(with = "crate::lotus_json")]
+    pub code: Cid,
+    #[schemars(with = "LotusJson<RawBytes>")]
+    #[serde(with = "crate::lotus_json")]
+    pub input: RawBytes,
+    pub caller: [u8; 20],
+    #[schemars(with = "LotusJson<TokenAmount>")]
+    #[serde(with = "crate::lotus_json")]
+    pub value: TokenAmount,
+}
+
+macro_rules! impl_evm_delegate_call_params_lotus_json {
+($($version:literal),+) => {
+        $(
+            paste! {
+            impl HasLotusJson for fil_actor_evm_state::[<v $version>]::DelegateCallParams {
+                type LotusJson = EVMDelegateCallParamsLotusJson;
+
+                #[cfg(test)]
+                fn snapshots() -> Vec<(serde_json::Value, Self)> {
+                    vec![(
+                        json!({
+                            "Code": "bafy2bzaceaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa",
+                            "Input": "ESIzRFU=",
+                            "Caller": [0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],
+                            "Value": "0"
+                        }),
+                        Self {
+                            code: Cid::default(),
+                            input: hex::decode("1122334455").unwrap(),
+                            caller: fil_actor_evm_state::evm_shared::[<v $version>]::address::EthAddress([0; 20]),
+                            value: TokenAmount::from_atto(0).into(),
+                        },
+                    )]
+                }
+
+                fn into_lotus_json(self) -> Self::LotusJson {
+                    EVMDelegateCallParamsLotusJson {
+                        code: self.code,
+                        input: self.input.into(),
+                        caller: self.caller.0,
+                        value: self.value.into(),
+                    }
+                }
+                fn from_lotus_json(lotus_json: Self::LotusJson) -> Self {
+                    Self {
+                        code: lotus_json.code,
+                        input: lotus_json.input.into(),
+                        caller: fil_actor_evm_state::evm_shared::[<v $version>]::address::EthAddress(lotus_json.caller),
+                        value: lotus_json.value.into(),
+                    }
+                }
+            }}
+        )+
+    };
+}
+
+impl_evm_delegate_call_params_lotus_json!(10, 11, 12, 13, 14, 15, 16);
+
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize, JsonSchema)]
+#[serde(rename_all = "PascalCase")]
+pub struct GetStorageAtParamsLotusJson {
+    #[schemars(with = "Vec<u8>")]
```

Comment:
I think schemars attribute can be omitted here

Reply:
removed